shell/shared/db.js

```diff
@@ -86,6 +86,46 @@ Grains = new Mongo.Collection("grains");
 //   publicId:  An id used to publicly identify this grain. Used e.g. to route incoming e-mail and
 //       web publishing. This field is initialized when first requested by the app.
 
+RoleAssignments = new Mongo.Collection("roleAssignments");
+// Edges in the permissions sharing graph.
+//
+// To share a permission with another user is to declare "if I have this permission, then so should
+// the recipient". A bundle of such declarations by a single sharer directed at a single recipient
+// for a single grain is called a "role assignment". A grain's owner always has every permission,
+// but permissions for other users must be computed from this collection.
+//
+// Every role assignment implicitly shares the special "can view grain" permission, which dictates
```

Comment:
I want to describe it as a permission because its value will be computed that same way as for other permissions -- this is, flowing from the grain owner along RoleAssignments.

I think "can tell that the grain exists" is a weaker permission than I have in mind here. I'm thinking more like "can get a UiSession".